packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.tsx

```diff
@@ -96,6 +105,19 @@ const Navigation = (props: OwnProps & Props) => {
         Success: (coinList) => <Success coinList={coinList} />
       })}
       <Divider margin='0 16px 8px 16px' />
+      {walletDebitCardEnabled && (
+        <LinkContainer to='/debitCard' activeClassName='active'>
+          <MenuItem data-e2e='debitCardLink'>
+            <DebitCardNavItem />
+            <NewCartridge>
+              <FormattedMessage
+                id='layouts.wallet.menuleft.navigation.debit_card.order'
+                defaultMessage='ORDER'
```

Comment:
we cant rely on translators to keep the same casing.  
1). name key to `copy.order`, value being `order`
2). use CSS `text-transform: uppercase` in NewCartridge

Reply:
done